crates/pera-node/src/lib.rs

```diff
@@ -1290,7 +1291,14 @@ impl PeraNode {
                     .await?;
             }
         }
-
+        epoch_store
+            .set_signature_mpc_manager(SignatureMPCManager::new(
+                Arc::new(consensus_adapter.clone()),
+                Arc::downgrade(&epoch_store),
+                config.max_active_mpc_instances,
```

Comment:
This might become confusing in the future.
The config lists `max_active_mpc_instances`, suggesting that this count is the max active instances over all. In this line, _all_ that capacity is assigned to the `SignatureMPCManager`. Right now, that is the only MPC Manager, but IIRC, there could be more. If that is the case / that happens, it will most likely happen that this capacity is also assigned to them, thus yielding double the capacity.

I, therefore, propose renaming the `Config` parameter to `max_active_signature_mpc_instances`. If new managers are introduced in the future, it is clear that this value cannot be reused there.

Reply:
this resolves it